src/test/storage/table_test.cpp

```diff
@@ -12,63 +12,60 @@
 
 namespace opossum {
 
-// class StorageTableTest : public BaseTest {
-//  protected:
-//   void SetUp() override {
-//     t.add_column("col_1", "int");
-//     t.add_column("col_2", "string");
-//   }
+class StorageTableTest : public BaseTest {
```

Comment:
Test(s) for `Table#column_names()` and the const overload of `Table#get_chunk()` are missing.